public_html/wp-content/themes/wordcamp-central-2012/template-home.php

```diff
@@ -105,7 +107,23 @@
 			</a>
 		</div><!-- .wc-news -->
 
-		<div class="wc-tweets last">
+		<div class="wc-global-sponsors last">
+			<h3><strong>Some Of Our Sponsors</strong></h3>
+
+			<?php
+			echo wp_kses_post( MES_Block\render( array(
+				'number'      => 3,
+				'region_id'   => 'all',
+				'level_id'    => 3040794,
+				'image_align' => 'center',
+				'image_width' => 300,
+			) ) );
+			?>
+
+			<a href="<?php echo esc_attr( get_permalink( get_page_by_path( 'global-community-sponsors' ) ) ); ?>" class="more">More sponsors &rarr;</a>
+		</div>  <!-- .wc-global-sponsors -->
+
+		<!--<div class="wc-tweets last">
```

Comment:
Can we delete this code, rather than commenting it out?

Reply:
91c2d44b